scripts/run-qemu-ota

```diff
@@ -13,6 +13,9 @@ def main():
     parser = ArgumentParser(description='Run meta-updater image in qemu')
     parser.add_argument('imagename', default='core-image-minimal', nargs='?',
                         help="Either the name of the bitbake image target, or a path to the image to run")
+    parser.add_argument('--uboot', default='no',
```

Comment:
The default here should be "yes" so as to not break backwards compatibility. Also, it appears to me that this impacts much more than just u-boot, so maybe the flag should actually reference OSTree. Maybe better to name it `--ostree` and again default it to "yes". Or just use `--nonostree` as a flag.

Reply:
Also worth pointing out: I've been intending to add a parameter to this script to allow supplying a custom u-boot rom, and the nicest thing to do would be to called it `--uboot`. I might try to make a PR today. It would be another argument in favor of renaming this option.

Reply:
See https://github.com/advancedtelematic/meta-updater/pull/638 for my draft.

Reply:
Yes, I see, I think this is two separate things, one option allows as to specify u-boot.rom file and second allow as to disable booting using u-boot at all. 
So my proposal is to have two options,  and both of them with the default values:
1. `--uboot=None` (allows to specify custom  ROM file)
2. `--uboot-enable=yes` (disable\enable,  ignoring any other option related to u-boot)

`--ostree` or `--nonostree` is ok but those names just do not represent what exactly we do, so my vote for (`--uboot, --uboot-enable`)